packages/react-internal/src/components/pickers/BasePicker.tsx

```diff
@@ -274,15 +274,14 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends React.Componen
           input: css('ms-BasePicker-input', legacyStyles.pickerInput, inputProps && inputProps.className),
           screenReaderText: legacyStyles.screenReaderOnly,
         };
-
     return (
       <div ref={this.root} className={classNames.root} onKeyDown={this.onKeyDown} onBlur={this.onBlur}>
         <FocusZone
           componentRef={this.focusZone}
           direction={FocusZoneDirection.bidirectional}
           shouldEnterInnerZone={this._shouldFocusZoneEnterInnerZone}
-          role={'combobox'}
-          id={this._ariaMap.combobox}
+          role={this.canAddItems() ? 'combobox' : undefined}
```

Comment:
nit:
let's have `const canAddItems = this.canAddItems()` and use `canAddItems` here and below instead of calling it multiple times